frontend/server/src/main/java/com/amazonaws/ml/mms/wlm/WorkerThread.java

```diff
@@ -97,6 +103,17 @@ public WorkerThread(
         startTime = System.currentTimeMillis();
         lifeCycle = new WorkerLifeCycle(configManager, model);
         replies = new ArrayBlockingQueue<>(1);
+        workerLoadTime =
+                new Metric(
+                        "W-"
+                                + this.port
+                                + '-'
+                                + model.getModelName()
+                                        .substring(0, Math.min(model.getModelName().length(), 25)),
+                        Long.toString(System.currentTimeMillis()),
```

Comment:
Stri
```suggestion
                        String.valueOf(System.currentTimeMillis()),
```

Reply:
I thought it was better to use the objects-type's toString method. Why is this better?

Reply:
String.valueOf() has all the overload for most of common types.
Use String.valueOf() will be uniformed for all data types.